src/components/root/styled.ts

```diff
@@ -1,14 +1,10 @@
 import styled from 'styled-components';
 
-import Common from '../common/styled';
-
-const Root = styled(Common.Container)`
+const Root = styled.div`
   display: flex;
   flex-direction: column;
   flex: 1 0 auto;
   height: calc(100% - 74px);
-  margin: 0 auto;
-  margin-top: 74px;
```

Comment:
Vi hadde `margin-top` for at Root ikke skulle krasje med header i visse situasjoner. Hvorfor fjerne?

Reply:
Ny header har ikke "position: fixed", margin-top ble da unødvendig.